### PR TITLE
Default bitstream support

### PR DESCRIPTION
**Pull request by mkurc-ant**
This PR adds support for default bitstream that is then modified by FASM features for a design:
 - Added support for implicit inverted FASM features that have `NOT_` prefix in their last part
 - Added support for loading a default bitstream either from a specified file or from FASM database (if present)
 - Added utility scripts for generating default bitstream for `qlf_k4n8`
 - Updated and added tests

### CODE REVIEW

File: make_qlf_k4n8_default_fasm.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    logical_tile_io_mode_io__9.logical_tile_io_mode_physical__iopad_0.logical_tile_io_mode_physical__iopad_mode_default__pad_0.IO_QL_CCFF_mem.mem_out,\
+    logical_tile_io_mode_io__10.logical_tile_io_mode_physical__iopad_0.logical_tile_io_mode_physical__iopad_mode_default__pad_0.IO_QL_CCFF_mem.mem_out,\
+    logical_tile_io_mode_io__11.logical_tile_io_mode_physical__iopad_0.logical_tile_io_mode_physical__iopad_mode_default__pad_0.IO_QL_CCFF_mem.mem_out,\
+    logical_tile_io_mode_io__12.logical_tile_io_mode_physical__iopad_0.logical_tile_io_mode_physical__iopad_mode_default__pad_0.IO_QL_CCFF_mem.mem_out,\
+    logical_tile_io_mode_io__13.logical_tile_io_mode_physical__iopad_0.logical_tile_io_mode_physical__iopad_mode_default__pad_0.IO_QL_CCFF_mem.mem_out,\
+    logical_tile_io_mode_io__14.logical_tile_io_mode_physical__iopad_0.logical_tile_io_mode_physical__iopad_mode_default__pad_0.IO_QL_CCFF_mem.mem_out,\
+    logical_tile_io_mode_io__15.logical_tile_io_mode_physical__iopad_0.logical_tile_io_mode_physical__iopad_mode_default__pad_0.IO_QL_CCFF_mem.mem_out"
```
Comment (acomodi) [inline make_qlf_k4n8_default_fasm.sh]: Can this be generated somehow? Or maybe defined in an external file and read from there

Comment (mkurc-ant) [reply make_qlf_k4n8_default_fasm.sh]: Hmm, The actual bit name is unique but the only difference between them is a single number. I can make a for loop that generates all the names from a single template.

Comment (mkurc-ant) [reply make_qlf_k4n8_default_fasm.sh]: Ok, fixed.

**Review by sharmaln — APPROVED** [review]
Looks good to me.

**Review by tpagarani — APPROVED** [review]

**Pull request merged**